remote-access/web-server/nginx.md

```diff
@@ -4,6 +4,19 @@ NGINX (pronounced *engine x*) is a popular lightweight web server application yo
 
 Like Apache, NGINX can serve HTML files over HTTP, and with additional modules can serve dynamic web pages using scripting languages such as PHP.
 
+## Refresh database of available packages
+
+Ensure that the package manager has up-to-date information about which packages are available:  
+
+```bash
+sudo apt-get update
+```
```

Comment:
We are suggesting the use of apt rather than apt-get nowadays, but apart from that, LGTM.